BoxContentSDK/BoxContentSDK/Requests/BOXFileUploadNewVersionRequest.m

```diff
@@ -78,7 +79,7 @@ - (BOXAPIOperation *)createOperation
         }
 
         operation.uploadMultipartCopyFilePath = self.uploadMultipartCopyFilePath;
-        operation.associateId = self.associateId;
+        operation.associateId = self.associateID;
```

Comment:
There are many many of these rename changes.  I suggest that (if it's possible) you put a prelim PR for the rename so the bug-fix PR is more focused.

Reply:
added https://github.com/box/box-ios-sdk/pull/649